### DISCUSSION TITLE
Holy Cow

### DISCUSSION THREAD

**Discussion by ketchup57** [Security-Onion-Solutions/securityonion#10876] [2.4]
Can I just say this 2.4 version is absolutely amazing to look at with these new dashboards. Definitely going to be tinkering around trying to wrap my head around things.
I come from a network engineering side of the house so wrapping my head around the back end will be tough in some areas. But holy smokes. Loving this. 

Trying not start multiples, but is it possible to have integration modules stream from the same input like udp 514?
Reason being everything logs into this port but the difference in how the modules read is based on the logging facility header code.

**Comment by dougburks** [comment]
> Can I just say this 2.4 version is absolutely amazing to look at with these new dashboards. Definitely going to be tinkering around trying to wrap my head around things.
> I come from a network engineering side of the house so wrapping my head around the back end will be tough in some areas. But holy smokes. Loving this.

Thanks, glad you like it!

> Trying not start multiples, but is it possible to have integration modules stream from the same input like udp 514?
> Reason being everything logs into this port but the difference in how the modules read is based on the logging facility header code.

Assuming you're referring to your other discussion at https://github.com/Security-Onion-Solutions/securityonion/discussions/10843, please continue that discussion there rather than starting multiple discussions.
